examples/platforms/simulation/radio.c

```diff
@@ -1016,6 +1047,56 @@ otError otPlatRadioGetCoexMetrics(otInstance *aInstance, otRadioCoexMetrics *aCo
 }
 #endif
 
+uint64_t otPlatRadioGetNow(otInstance *aInstance)
+{
+    OT_UNUSED_VARIABLE(aInstance);
+
+    return otPlatTimeGet();
+}
+
+#if OPENTHREAD_CONFIG_MAC_CSL_RECEIVER_ENABLE
+static otError updateIeData(otInstance *aInstance)
+
+{
+    OT_UNUSED_VARIABLE(aInstance);
+
+    otError error  = OT_ERROR_NONE;
+    uint8_t offset = 0;
+
+#if OPENTHREAD_CONFIG_MAC_CSL_RECEIVER_ENABLE
+    if (sCslPeriod > 0)
+    {
+        memcpy(sAckIeData, sCslIeHeader, IE_HEADER_SIZE);
```

Comment:
`sAckIeData` is not used in this PR?

Reply:
Ah yes. This could actually be removed because we use `otMacFrameGenerateXXXAck` to get ACKs in simulation.

Reply:
Correction: It's required as parameters to be passed to `otMacFrameGenerateEnhAck`. It wasn't called in last commit and it's fixed now.